src/components/textbox/textbox.component.tsx

```diff
@@ -82,7 +82,7 @@ export interface CommonTextboxProps
   leftChildren?: React.ReactNode;
   /** Label content */
   label?: string;
-  /** [Legacy] Inline label alignment */
+  /** Label alignment */
```

Comment:
**suggestion**: If these changes are made, they should also be applied to `Textarea` to ensure consistency

Reply:
Fixed in 6af9fda57cb95af3f8d40efc06fba858f9550015